### DISCUSSION TITLE
My Meetings are not showing up on the app

### DISCUSSION THREAD

**Discussion by RMcD47** [code4recovery/12-step-meeting-list#1077] [General]
I uploaded my spreasheet & the meetings show up in the wordpress app on my site, but I'm not seeing them on the meetings app. Is there a format for the import that I may be messing up. Is it better to delete all the meetings and try again (I feel like I read somewhere that you were not suppposed to do that).

Thank you
Roxanna
[12-step-meeting-list.3.14.12 (1).zip](https://github.com/code4recovery/12-step-meeting-list/files/10728408/12-step-meeting-list.3.14.12.1.zip)

**Comment by tim-rohrer** [comment]
Hi Roxanna,

Is this a new site you're setting up?

If meetings are showing up on your website, and everything looks good there, two more steps are typically needed to get meetings into Meeting Guide.

1. You need to authorize your website (TSML) to connect to Meeting Guide (Meetings -> Settings -> Authorized Apps -> Add).
2. You need to register with Meeting Guide (https://www.aa.org/meeting-guide-app, then scroll a bit and you'll see a link about getting connected).

If I'm misunderstanding your issue, please provide a link to your site and I might be able to help look.

However, in general, if meetings look good on your site, and you've previously setup with Meeting Guide, then you'll need to contact them if meetings aren't showing up there.

Let us know.

Tim

**Reply by RMcD47** [reply]
Thank you. I do not see the option "Authorized Apps" in my settings section of the Meetings plugin. Is this still an option.
I registered with the Meeting guide but when I clicked on the "Public Data Source" I was given the list of meetings in a code format. I don't think this is what I was supposed to provide. Can you please advise? Thank you.

**Reply by joshreisner** [reply]
yes, give app support the address of the page with the meetings in code format. that is called a JSON feed. they are looking for the URL of that page

**Reply by RMcD47** [reply]
Can you tell me how to add the page to my menus? It doesn't show up in my page list only on website.com/meetings/

**Reply by kiyote33** [reply]
Hi @RMcD47, you might try one of two options in WordPress Admin: 1) Add a custom link on the /Appearance/Menus page called Meetings with a URL = /meetings, or 2) just add a blank Meetings page (and it will automajically get populated with your 12 Step Meeting List) if the setting to have root pages automatically added to your menu is turned on.

**Comment by kiyote33** [comment]
Hi @RMcD47,
On the Settings page under Feed Management, the default Feed Sharing setting is Open which allows access to your Meetings Feed without authorization. If you prefer to restrict access to your meetings feed, you have the option to change the Open Feed Sharing option to Restricted. An Authorized Apps setting then appears under the Feed Sharing setting which allows you to add the "Meeting Guide" to access your meetings feed.
Hope this helps...
